src/commonMain/kotlin/fr/acinq/lightning/crypto/sphinx/Sphinx.kt

```diff
@@ -295,11 +294,8 @@ data class DecryptedFailurePacket(val originNode: PublicKey, val failureMessage:
  * +----------------+----------------------------------+-----------------+----------------------+-----+
  * with failure message length + pad length = 256
  */
-@ThreadLocal
```

Comment:
I suppose we can remove all the `@ThreadLocal`

Reply:
Yes (it was there because of `FailurePacket.logger`)